site/src/components/card.md

```diff
@@ -199,5 +199,25 @@ Another available option is using the `.card-content` inner container, and placi
 		<a class="card-title">The title of your content item here.</a>
 		<p>Additional metadata can be placed here.</p>
 	</div>
+	<div class="card-footer">
+		<div class="card-footer-item">
+			<span class="color-success">
+				<span>Completed</span>
+				<span class="icon">
+					<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448" class="fill-current-color">
```

Comment:
Worth adding `aria-hidden` for safer copy-pasteability?

Reply:
Yes! done